src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -1024,9 +1024,11 @@ namespace SurfaceGeometry {
             state.dataSurface->SurfExtConvCoeffIndex(SurfNum) = 0;
             state.dataSurface->SurfTAirRef(SurfNum) = 0;
             state.dataSurface->SurfIntConvClassification(SurfNum) = ConvectionConstants::InConvClass::Invalid;
+            state.dataSurface->SurfIntConvClassificationRpt(SurfNum) = static_cast<int>(ConvectionConstants::InConvClass::Invalid);
             state.dataSurface->SurfIntConvHcModelEq(SurfNum) = 0;
             state.dataSurface->SurfIntConvHcUserCurveIndex(SurfNum) = 0;
             state.dataSurface->SurfOutConvClassification(SurfNum) = ConvectionConstants::OutConvClass::Invalid;
+            state.dataSurface->SurfOutConvClassificationRpt(SurfNum) = static_cast<int>(ConvectionConstants::OutConvClass::Invalid);
```

Comment:
I am curious if this will cause (corrective) diffs now compared to yesterday.

Reply:
Unknown. I just ran regressions locally (on Mac) and see zero difference between current develop and this branch. I don't have a way to run regressions on other platforms, so I too am curious if it helps.

Reply:
Well, the right 2 example files had diffs but this doesn't make any sense. The new variables SurfIn/OutConvClassificationRpt aren't acted on by anything, they just store the values of the old variables and report them. I wonder if the functions where the old variables are calculated are called every time step? Anyway, if it fixes diffs then so be it.